functions/Install-DbaWhoIsActive.ps1

```diff
@@ -136,8 +136,9 @@ function Install-DbaWhoIsActive {
             # Unpack
             # Unblock if there's a block
             if ($PSCmdlet.ShouldProcess($env:computername, "Unpacking zipfile")) {
-
-                Unblock-File $zipfile -ErrorAction SilentlyContinue
+                if ([Environment]::OSVersion.Platfrom -eq 'Win32NT') {
```

Comment:
I would prefer to use `$PSVersionTable.Platform` as that is what we are doing in other places in the module.

Reply:
I am basing this change/type off of the following article.. (https://powershell.org/2019/02/tips-for-writing-cross-platform-powershell-code/) $PSVersionTable.Platfrom returns null on 5.1

Reply:
[Environment]::OSVersion.Platfrom returns a value regardless of version.

Reply:
Although I just realized it is misspelled.. 😨

Reply:
https://github.com/sqlcollaborative/dbatools/blob/e8d440c0e09a4937b7301696cb3c7dba7128d705/dbatools.psm1#L11-L15

Reply:
I understand that is how it is currently being done, but maybe we could change the way we are doing it. [Environment]::OSVersion is supported back to .NET 1.1 and give a ubiquitous answer.. WinNT32 or Unix. It is just a little more concise. I guess that is why I like it. I have the spell change staged, but if you want the long form I can rewrite.

Reply:
My only issue with it is it takes longer to run...even longer in PS6+ than just using `$PSVersionTable`.

![image](https://user-images.githubusercontent.com/11204251/67147126-91944280-f257-11e9-8503-7ff3d24ef95e.png)

![image](https://user-images.githubusercontent.com/11204251/67147128-9527c980-f257-11e9-96c1-2d73c2ee1bb0.png)

Reply:
While that is just showing ms time, we start doing it everywhere in commands that we get to cross-platform and it starts adding up.

Reply:
So I have an alternative. We just use the internal function `Test-Windows`
```suggestion
                if (Test-Windows) {
```

Reply:
Cool.. that works too.. was about to suggest something like that. I am going to look at the other `Install-` cmdlets as well, so I knew this was going to come up again..